charts/nri-bundle/Chart.yaml

```diff
@@ -68,6 +68,9 @@ dependencies:
     condition: newrelic-pixie.enabled
     version: 2.0.6
 
+  # when bumping the pixie-operator-chart, also bumping the versions of
```

Comment:
nit: reword as `Keep the versions of pixie-operator-chart in sync with the CRD versions for`